site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-12-February-2022.adoc

```diff
@@ -0,0 +1,93 @@
+= Apache Cassandra Changelog #12
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: February 10, 2022
+:page-post-author: The Apache Cassandra Community
+:description: The Apache Cassandra Community
+:keywords: 
+
+image::blog/changelog_header.jpg[Apache Cassandra Changelog]
+Our monthly roundup of key activities and knowledge to keep the community informed.
+
+= Release Notes
+
+=== Release
+
+Available since 7th September, the latest release of Apache Cassandra is  https://www.apache.org/dyn/closer.lua/cassandra/4.0.1/[4.0.1^], (https://downloads.apache.org/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.asc[pgp^],  https://downloads.apache.org/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha256[sha256^], and https://archive.apache.org/dist/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha512[sha512^]). This was a rapid release to fix a https://issues.apache.org/jira/browse/CASSANDRA-16877[critical bug^] in Gossip on large clusters. Please read the https://gitbox.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=CHANGES.txt;hb=refs/tags/cassandra-4.0.1[release notes^] and let us know if you encounter any problems.
```

Comment:
```suggestion
Available since 7th September, the latest release of Apache Cassandra is  https://www.apache.org/dyn/closer.lua/cassandra/4.0.1/[4.0.1^], (https://downloads.apache.org/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.asc[pgp^],  https://downloads.apache.org/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha256[sha256^], and https://archive.apache.org/dist/cassandra/4.0.1/apache-cassandra-4.0.1-bin.tar.gz.sha512[sha512^]). This was a rapid release to fix a https://issues.apache.org/jira/browse/CASSANDRA-16877[critical bug^] in Gossip on large clusters. Please read the https://github.com/apache/cassandra/blob/cassandra-4.0.1/CHANGES.txt[release notes^] and let us know if you encounter any problems.
```